returnn/tf/util/data.py

```diff
@@ -824,16 +824,39 @@ def copy_with_time_dim_axis(self, time_dim_axis):
     assert self.time_dim_axis is not None
     return self.copy_move_axis(self.time_dim_axis, time_dim_axis)
 
-  def copy_move_axis(self, old_axis, new_axis, keep_other_axes_intact=False):
+  def _copy_template_translate_axes(self, translate_axis):
```

Comment:
This could be `copy_transpose`, and could be public, and also not just on template.